qdrant-landing/content/blog/qdrant-1.14.x.md

```diff
@@ -0,0 +1,269 @@
+                    }
+                }
+            ]
+        },
+        "defaults": {
+            "geo.location": { "lat": 48.137154, "lon": 11.576124 } // Munich
+        }
+    }
+}
+```
+
+You can tweak parameters like `target`, `scale`, and `midpoint` to shape how quickly the score decays over distance. This is extremely useful for local search scenarios, where location is a major factor but not the only factor.
+
+> This is a very powerful feature that allows for extensive customization. Read more about this feature in the [**Hybrid Queries Documentation**](/documentation/concepts/hybrid-queries/)
+
+## Incremental HNSW Indexing 
+![optimizations](/blog/qdrant-1.14.x/optimizations.jpg)
+
+Rebuilding an entire [**HNSW graph**](/documentation/concepts/indexing/#vector-index) every time new data is added can be computationally expensive. With this release, Qdrant now supports incremental HNSW indexing—an approach that extends existing HNSW graphs rather than recreating them from scratch.
+
+> This feature is designed to make indexing faster and more efficient when you’re only adding new points. It reuses the existing structure of the HNSW graph and appends the new data directly onto it. 
+
+That means much less time spent building and more time searching. Although this initial implementation currently only support upserts, it lays the groundwork for a more dynamic and performance-friendly indexing process. Especially for collections with frequent updates to payload values or growing datasets, incremental HNSW is a big step forward.
+
+> Note that deletes and updates will still trigger a full rebuild. Check out the [**indexing documentation**](/documentation/concepts/indexing/) to learn more. 
+
+## Faster Batch Queries
+![reranking](/blog/qdrant-1.14.x/gridstore.jpg)
+
+In this release, Qdrant introduces a major performance boost for [**batch search operations**](/documentation/concepts/search/#batch-search-api). Until now, the query batch API used a single thread per segment, which worked well—unless you had just one segment and a large batch of queries. In such cases, everything was processed on a single thread, significantly slowing things down. This scenario was especially common when using our [**Python client**](https://github.com/qdrant/qdrant-client), which is single-threaded by default.
+
+The new optimization changes that. Large query batches are now split into chunks, and each chunk is processed on a separate thread. 
+
+> This allows Qdrant to execute queries concurrently, even when operating over a single segment. 
+
+You will get much faster response times for large batches of queries. If you’re working with high-volume query workloads, you should notice a significant improvement in latency. Benchmark results show just how dramatic the difference can be.
+
+Using our [**bfb benchmarking tool**](https://github.com/qdrant/bfb), we populated a 1-segment collection and ran a request of 240 batch queries.
```

Comment:
Added 'basic test' as you might not call this very scientific. But since the difference is so dramatic its already good enough to prove there's a perf improvement. Not sure about my exact wording here though.

```suggestion
As a basic test, we populated a 1-segment collection using our [**bfb benchmarking tool**](https://github.com/qdrant/bfb), and ran a request of 240 batch queries.
```